.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+
+install:
+  - pip install -r requirements.txt
+  - git clone https://github.com/simphony/JYU-LB.git
+  - cd JYU-LB
+  - make
+  - cd ..
+  - mv JYU-LB/bin/jyu_lb_isothermal3D.exe .
```

Comment:
The JYU-LB simulation software is supposed to be operating system independent (in particular no GUI):
it uses only C++ standard library and OpenMP. So far compiled only with g++ ... my desktop has Windows,
hence I'm used to .exe (although I now use Cygwin for SimPhoNy work).

I will add the python setup.py line for install (with no full understanding about the ramifications).